java/src/jmri/implementation/DefaultStringIO.java

```diff
@@ -0,0 +1,50 @@
+package jmri.implementation;
+
+import jmri.JmriException;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Base implementation of the StringIO interface.
+ *
+ * @author Bob Jacobsen  (C) 2024
+ */
+public class DefaultStringIO extends AbstractStringIO {
+
+    public DefaultStringIO(@Nonnull String systemName) {
+        super(systemName);
+    }
+
+    public DefaultStringIO(@Nonnull String systemName, String userName) {
+        super(systemName, userName);
+    }
+
+    /**
+     * Sends the string to the layout.
+     * The string [u]must not be longer than the value of getMaximumLength()
+     * unless that value is zero. Some microcomputers have little memory and
+     * it's very important that this method is never called with too long strings.
+     *
+     * @param value the desired string value
+     * @throws jmri.JmriException general error when setting the value fails
+     */
```

Comment:
I suggest removing the comment and replace it with `/** {@inheritDoc} */` instead.

Reply:
Done, thanks.